realestatinator/views/estate_property_views.xml

```diff
@@ -1,8 +1,82 @@
 <?xml version="1.0"?>
 <odoo>
 	<record id="estate_property_model_action" model="ir.actions.act_window">
-	    <field name="name">Estate Property Model Action</field>
-	    <field name="res_model">estate_property</field>
-	    <field name="view_mode">kanban,form</field>
+		<field name="name">Estate Property Model Action</field>
+		<field name="res_model">estate_property</field>
+		<field name="view_mode">list,form</field>
+	</record>
+	<record id="estate_property_model_list" model="ir.ui.view">
+		<field name="name">estate_property_list</field>
```

Comment:
Guildines for views :
id="estate_property_view_search" = 'modelName_view_viewType'
and the name should be the same but using '.' instead of '_'
apply this everywhere (on all views - not actions) :)